fb_apt/README.md

```diff
@@ -25,9 +25,9 @@ Usage
 -----
 To install and configure APT include `fb_apt`, which will populate the 
 repository sources in `/etc/apt/sources.list` and update the package cache 
-every `node['fb_apt']['update_delay']` seconds (defaults to 86400) using the 
-`execute[apt-get update]` resource (which other cookbooks can suscribe to or 
-notify).
+during the run if it's old than `node['fb_apt']['update_delay']` seconds
+(defaults to 86400). This uses the `execute[apt-get update]` resource, which
+other cookbooks can suscribe to or notify as well.
```

Comment:
it's obvious if you read the code, but you should note that setting this to 0 is the "right" way to have it update on every run